tasks.gradle

```diff
@@ -0,0 +1,13 @@
+apply from: '../config.gradle'
+
+private void replaceVersionsInFile(File file) {
+    def content = file.text
+    content = content.replaceAll(~/sentinelVersion\s*=\s*".*"/, "sentinelVersion = \"${releaseConfig.version}\"")
```

Comment:
It might be a good idea to try and add a [comment](https://stackoverflow.com/questions/4823468/comments-in-markdown) in the readme that this script needs to be updated if someone updates the content there too.

Reply:
Good idea